corehq/apps/app_manager/models.py

```diff
@@ -5713,6 +5713,20 @@ def undo(self):
         app.save()
 
 
+class SQLGlobalAppConfig(models.Model):
+    choices = [(c, c) for c in ("on", "off", "forced")]
+
+    domain = models.CharField(max_length=255, null=False)
+    app_id = models.CharField(max_length=255, null=False)
+    app_prompt = models.CharField(max_length=32, null=True, choices=choices, default="off")
+    apk_prompt = models.CharField(max_length=32, null=True, choices=choices, default="off")
+    apk_version = models.CharField(max_length=32, null=True)
+    app_version = models.IntegerField(max_length=32, null=True)
+
+    class Meta(object):
+        unique_together = ('domain', 'app_id')
```

Comment:
I think that this could just be a `unique=True` on the app_id. That would be more strict than the app_id, domain uniqueness constraint. I think that app_ids are still unique to the entire HQ environment right?

Reply:
App ids are unique to the environment, yes, but I'm wary of adding code that depends on that assumption.